docs/explanation/reader-explanation.md

```diff
@@ -0,0 +1,274 @@
+                                   "/dat_mat_components/Temperature 1 [filt]/unit",
+                                   "/dat_mat_components/Temperature 1 [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 1 X [filt]/value",
+                                   "/dat_mat_components/LI Demod 1 X [filt]/unit",
+                                   "/dat_mat_components/LI Demod 1 X [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 1 Y [filt]/value",
+                                   "/dat_mat_components/LI Demod 1 Y [filt]/unit",
+                                   "/dat_mat_components/LI Demod 1 Y [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 2 X [filt]/value",
+                                   "/dat_mat_components/LI Demod 2 X [filt]/unit",
+                                   "/dat_mat_components/LI Demod 2 X [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 2 Y [filt]/value",
+                                   "/dat_mat_components/LI Demod 2 Y [filt]/unit",
+                                   "/dat_mat_components/LI Demod 2 Y [filt]/metadata"]}
+}
+```
+**NOTES**
+
+- Each key is nevigating the NeXus concept (e.g. `/ENTRY[entry]/INSTRUMENT[instrument]/piezo_config/active_calib` key nevigates `ENTRY` -> `INSTRUMENT` -> `piezo_config` -> `active_calib` field in `NXsts` application definition.) in NeXus application definition.  
+- Value denoted by `@eln` refers the data must come from eln (user provided), but can be changed if rwa file contains that piece of data. 
+- To update (if needed) the config file, a set of rules needs to be followed:
+  - The dictionary in config files have the following meaning:
+    ```
+    "/ENTRY[entry]/INSTRUMENT[instrument]/lock_in/harmonic_order_N": {"D1": {"value": "/Lock-in/Harmonic D1/value"},
+                                                                      "D2": {"value": "/Lock-in/Harmonic D2/value"}},
+    ```
+    Here, the part `N` in field `harmonic_order_N`, can be considered as the name of dimensions, can be replaced by `D1` and `D2` to  write two `harmonic_order` and can be extended for further dimensions.
+  - List for the same concept
+    ```
+    "/ENTRY[entry]/INSTRUMENT[instrument]/piezo_config/active_calib": ["/Piezo Configuration/Active Calib.",
+                                                                       "/Piezo Calibration/Active Calib."],
+    ```
+    For different type of software versions the raw data path could be different for the same concept. For example, Nanonis software `generic 5e` has `/Piezo Configuration/Active Calib.` and generic 4.5 has `/Piezo Calibration/Active Calib.` for the same concept `/ENTRY[entry]/INSTRUMENT[instrument]/piezo_config/active_calib`.
+  - In the config file, the concept that takes data from ELN is denoted by `@eln`. Otherwise, data will come from experimental raw files.
```

Comment:
```suggestion
  - In the config file, concepts that take data from the ELN are denoted by `@eln`. Otherwise, data will come from experimental raw files.
```

Reply:
Resolved!